common/rfb/CConnection.h

```diff
@@ -121,6 +133,21 @@ namespace rfb {
 
     // Methods to be overridden in a derived class
 
+    /*!
+     * \brief getUserPasswd gets the username and password.
+     *        This might involve a dialog, getpass(), etc.
+     * \param secure: Indicates whether this connection is secure
+     * \param user: user name buffer pointer. the pointer may be nullptr,
+     *              in which case no user name will be retrieved.
+     * \param password: password buffer pointer.
+     */
```

Comment:
I'm afraid these documentation tags are not something we use. Please keep the existing style.